### ISSUE TITLE
[BUG] Add evidence URL as option when creating evidence

### ISSUE THREAD

**Issue by chrisdburr** [alan-turing-institute/AssurancePlatform#500]
As highlighted in #442 when a user adds an evidence element, there is no option to add a URL. It requires reselecting the evidence element after creation. 

Please can we add this field during element creation?

**Comment by RichGriff** [comment]
Evidence url now available on the evidence create form

![Image](https://github.com/alan-turing-institute/AssurancePlatform/assets/28445922/054b211c-e74f-4ffa-8231-f4cd14a39fe5)

**Comment by RichGriff** [comment]
In `staging` - @chrisdburr can you please review, thanks.

**Comment by chrisdburr** [comment]
Sorry, reopening as I have just found a bug while reviewing another issue. 

If the user leaves the URL field blank and then clicks the 'add' button, nothing happens and the button is left selected. You can't then add a URL subsequently.

https://github.com/alan-turing-institute/AssurancePlatform/assets/63010234/0f03dc3e-3a49-4c9c-a74b-cdac86b8c0dd

I suggest we allow the users to leave the URL blank rather than enforcing this. However, we should make sure that a NULL value is added to the DB.

**Comment by RichGriff** [comment]
Corrected the api to allow for `Null` value for the `URL` when creating `Evidence`.

**Comment by RichGriff** [comment]
In PR waiting on review

**Comment by RichGriff** [comment]
In staging

**Comment by chrisdburr** [comment]
Thanks, @RichGriff. This seems to be working on my end.
